inscriptions/ISic003203.xml

```diff
@@ -189,14 +194,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#FP">
+                <ab>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/> <persName type="attested"><name type="nomen"><supplied reason="lost">Ma</supplied>nliae</name></persName> <persName type="attested"><name type="praenomen"><expan><abbr>T</abbr><ex>iti</ex></expan></name></persName> <w><expan><abbr>f</abbr><ex>iliae</ex></expan></w>
+                    <lb n="2"/><supplied reason="lost"><w>et</w></supplied> <persName type="attested"><supplied reason="lost"><name type="praenomen">Marci</name></supplied> <name type="nomen"><supplied reason="lost">Ro</supplied>sci</name> <persName type="attested"><name type="praenomen"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <w><expan><abbr>f</abbr><ex>ili</ex></expan></w> <orgName type="tribus" nymRef="#Quirina"><expan><abbr>Qui</abbr><ex>rina</ex></expan> <supplied reason="subaudible"><w lemma="tribus">tribu</w></supplied></orgName> 
+                    <lb n="3"/><name type="cognomen">Cironni</name></persName> <w>uxori</w> <supplied reason="omitted"><w>piissimae</w></supplied> 
+                    <lb n="4"/><persName type="attested"><name type="praenomen"><expan><abbr>M</abbr><ex>arcus</ex></expan></name> <name type="nomen">Roscius</name> <persName type="attested"><name type="praenomen"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <g ref="#interpunct">·</g> <w><expan><abbr>f</abbr><ex>ilius</ex></expan></w> <g ref="#interpunct">·</g>
+                        <lb n="5"/><orgName type="tribus" nymRef="#Quirina"><expan><abbr>Qui</abbr><ex>rina</ex></expan> <supplied reason="subaudible"><w lemma="tribus">tribu</w></supplied></orgName> <name type="cognomen">Cironnus</name></persName>
+                    <lb n="6"/><w><expan><abbr>d</abbr><ex>icavit</ex></expan></w> <w><expan><abbr>d</abbr><ex>e</ex></expan></w> <w><expan><abbr>s</abbr><ex>ua</ex></expan></w> <w><expan><abbr>p</abbr><ex>ecunia</ex></expan></w>
```

Comment:
i'm sure this should be d(edicavit) not d(icavit), despite what Libertini writes. perhaps add the hedera (#ivy-leaf) between the first and second D also, as it is clear in the photograph